test/sdk.test.ts

```diff
@@ -0,0 +1,45 @@
+import { CapacitorOptions } from '../src';
+import { DEFAULT_OPTIONS_MOBILE, DEFAULT_OPTIONS_WEB, init } from '../src/sdk';
+import { NATIVE } from '../src/wrapper';
+
+jest.mock('../src/wrapper', () => {
+  return {
+    NATIVE: {
+      platform: 'web',
+      initNativeSdk: jest.fn(),
+    },
+  };
+});
+
+describe('SDK Init', () => {
+  it('Uses default web options on web', () => {
+    NATIVE.platform = 'web';
+
+    init({ dsn: '' }, (finalOptions: CapacitorOptions) => {
+      expect(finalOptions.enableNative).toBe(false);
+      expect(finalOptions).toEqual(
+        expect.objectContaining(DEFAULT_OPTIONS_WEB),
+      );
+    });
+  });
+
+  it('Uses default mobile options on ios', () => {
+    NATIVE.platform = 'ios';
+
+    init({ dsn: '' }, (finalOptions: CapacitorOptions) => {
+      expect(finalOptions).toEqual(
+        expect.objectContaining(DEFAULT_OPTIONS_MOBILE),
+      );
+    });
+  });
+
+  it('Uses default mobile options on android', () => {
+    NATIVE.platform = 'android';
+
+    init({ dsn: '' }, (finalOptions: CapacitorOptions) => {
+      expect(finalOptions).toEqual(
+        expect.objectContaining(DEFAULT_OPTIONS_MOBILE),
+      );
+    });
+  });
+});
```

Comment:
Let's table test this 👀 

```ts
// [string, platform, options]
const table: Array<[string, string, CapacitorOptions]> = [
  ['Uses default web options on web', 'web', DEFAULT_OPTIONS_WEB],
  ['Uses default mobile options on ios', 'ios', DEFAULT_OPTIONS_MOBILE],
  ['Uses default mobile options on android', 'android', DEFAULT_OPTIONS_MOBILE],
];

it.each(table)('%s', (...test) => {
  NATIVE.platform = test[1];
  init({ dsn: '' }, (finalOptions: CapacitorOptions) => {
    expect(finalOptions.enableNative).toBe(false);
    expect(finalOptions).toEqual(expect.objectContaining(test[2]));
  });
});
```

Reply:
Oh I did not know about this, would've been handy in lots of places lol